dist/LookingGlassConfig.d.ts

```diff
@@ -212,14 +234,14 @@ export declare class LookingGlassConfig extends EventTarget {
     get aspect(): number;
     get tileWidth(): number;
     get framebufferWidth(): number;
-    get quiltWidth(): 5 | 8;
-    get quiltHeight(): 6 | 9;
+    get quiltWidth(): 5 | 7 | 8 | 11;
```

Comment:
This means that the user must use one of these options, correct? Is there a way for them to have a fully custom tile arrangement for their quilt?

Reply:
that is correct, we currently set that based on the device. 

I'll update it so they can provide an override value.